riak_test/tests/stats_test.erl

```diff
@@ -95,12 +64,100 @@ confirm_initial_stats(StatData) ->
                          "riakc_put_block",
                          "riakc_delete_block_constrained"
                         ]],
-
-    lager:debug("~p", [proplists:get_value(<<"request_pool_workers">>, StatData)]),
+    ?assertEqual(1, proplists:get_value(<<"velvet_create_user_in_one">>, StatData)),
     ?assertEqual(rtcs:request_pool_size() - 1,
                  proplists:get_value(<<"request_pool_workers">>, StatData)),
     ?assertEqual(rtcs:bucket_list_pool_size(),
-                 proplists:get_value(<<"bucket_list_pool_workers">>, StatData)).
+                 proplists:get_value(<<"bucket_list_pool_workers">>, StatData));
+
+confirm_initial_stats(stanchion, UserConfig, Port) ->
+    Stats = query_stats(stanchion, UserConfig, Port),
+    lager:debug("length(Stats) = ~p", [length(Stats)]),
+    ?assert(160 < length(Stats)),
+    [begin
+         StatKey = list_to_binary(StatType ++ "_one"),
+         lager:debug("StatKey: ~p~n", [StatKey]),
+         ?assert(proplists:is_defined(StatKey, Stats)),
+         Value = proplists:get_value(StatKey, Stats),
+         ?assertEqual(0, Value)
+     end || StatType <- ["bucket_create",
+                         "bucket_delete",
+                         "bucket_put_acl",
+                         "riakc_get_cs_bucket",
+                         "riakc_delete_cs_bucket",
+                         "riakc_get_cs_user",
+                         "riakc_list_all_manifest_keys",
+                         "riakc_list_all_user_keys"
+                        ]],
+    confirm_stat_count(Stats, <<"user_create_one">>, 1),
+    confirm_stat_count(Stats, <<"riakc_put_cs_user_one">>, 1),
+
+    ?assert(proplists:is_defined(<<"memory_atom">>, Stats)),
+    ?assert(proplists:is_defined(<<"sys_process_count">>, Stats)),
+    ?assert(proplists:is_defined(<<"webmachine_mochiweb_active_sockets">>, Stats)),
```

Comment:
I would prefer adding `waiting_time_(95|99|100|mean|median)`  because they're most important items in Stanchion.